SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestSingleServerDynamicComponents/TestDynamicComponentActor.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDynamicComponentActor.h"
+#include "TestDynamicComponent.h"
+
+#include "Net/UnrealNetwork.h"
+
+ATestDynamicComponentActor::ATestDynamicComponentActor()
+{
+	bReplicates = true;
+	bAlwaysRelevant = true;
+}
+
+void ATestDynamicComponentActor::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	Super::GetLifetimeReplicatedProps(OutLifetimeProps);
+
+	DOREPLIFETIME(ATestDynamicComponentActor, OnSpawnComponent);
+	DOREPLIFETIME(ATestDynamicComponentActor, PostInitializeComponent);
+	DOREPLIFETIME(ATestDynamicComponentActor, LateAddedComponent);
+}
+
+void ATestDynamicComponentActor::PostInitializeComponents()
+{
+	Super::PostInitializeComponents();
+
+	if (HasAuthority() && !PostInitializeComponent)
```

Comment:
```suggestion
	if (HasAuthority() && PostInitializeComponent != nullptr)
```
Nit

Reply:
Yeah you have mentioned this to me before at some point, forgot about it though, will change it, thank you!